main.py

```diff
@@ -1,7 +1,31 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre_name in genres:
+        Genre.objects.create(name=genre_name)
+
+    first_name = ["George", "Kianu", "Scarlett", "Will", "Jaden", "Scarlett"]
+    last_name = ["Klooney", "Reaves", "Keegan", "Smith", "Smith", "Johansson"]
+    if len(first_name) != len(last_name):
+        raise ValueError("Value error")
```

Comment:
Why did you add this check ? You can create just one list with tuples inside . For example [(first_name , last_name ), ...] . It will be more cleaner